images/runtime/node/14/base.buster.Dockerfile

```diff
@@ -20,6 +20,7 @@ ARG DEBIAN_FLAVOR
 ENV NODE_VERSION ${NODE14_VERSION}
 ENV NPM_CONFIG_LOGLEVEL info
 
+ARG BUILD_DIR=/tmp/oryx/build
```

Comment:
Could you please share more info about this change? Why do we need to add this now?

Reply:
This line used to exist in this file (since it's used to reference the `__nodeVersions.sh` script a few lines down), but I think it was accidentally removed in a previous PR, so I'm just adding it back so this Dockerfile (and the others where this change was made) can successfully build.

Reply:
[Here](https://github.com/microsoft/Oryx/commit/cb7d9947017fe217a8664a401548f5080e472a36#diff-fe86717626fd70be5d55410ea401debfdb9ce42d571a8b4053d285bbf25ad2e4L24) is the commit where it was accidentally removed for context